shell/pom.xml

```diff
@@ -63,7 +63,16 @@
       <groupId>jline</groupId>
       <artifactId>jline</artifactId>
    </dependency>
-
+      <dependency>
+          <groupId>com.fasterxml.jackson.dataformat</groupId>
+          <artifactId>jackson-dataformat-yaml</artifactId>
+          <version>2.9.9</version>
+      </dependency>
+      <dependency>
+          <groupId>com.fasterxml.jackson.datatype</groupId>
+          <artifactId>jackson-datatype-jsr310</artifactId>
+          <version>2.10.0.pr1</version>
+      </dependency>
```

Comment:
At least one of these is required. I don't think both are. Can you double check?